Opdracht gemaakt

Sadotu opened the pull request.

Sadotu committed:
Opdracht gemaakt

newz1111 commented on functions-and-scope.js:
```diff
@@ -64,21 +88,34 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // ---- Verwachte uitkomst: 9
-
+function highestGrade(grades) {
+    let highestGrade = 0
```
Het is geen goed voorbeeld, maar ik wil het je wel meegeven:
Wanneer je een array meekrijgt ([-1, -3, -2]) met alleen maar negatieve getallen, dan zal de functie altijd `0` teruggeven. Het is daarom verstandig om altijd de eerste waarde in de array te pakken:
```suggestion
    let highestGrade = gradese[0];
```

newz1111 reviewed (commented):
Goed gedaan! Ik heb een comment voor je achtergelaten 😉

Sadotu committed:
Update functions-and-scope.js

Ik zal hem aanpassen omdat het handig is om code te schrijven die makkelijker te herbruiken is.

Co-authored-by: Zwen1111 <zwen12@msn.com>
